test/integration/cases_test.go

```diff
@@ -255,6 +330,30 @@ var _ = Describe("JobSet metrics", func() {
 				jobset.recovery_count.WithValue(1),
 			)
 		})
+
+		It("should watch a jobset with a single replicated job", func() {
+			Expect(k8sClient.Create(ctx, jobsetSingleJob)).To(Succeed())
```

Comment:
We have 2 patterns in this test now: 1 create jobset inline, 2 reference jobsets created elsewhere via a variable. Lets consolidate to a single jobset builder func.

Reply:
I simplified in 717d1ad without a builder. If we keep adding tests, perhaps we revisit having a builder func then?